src/main/scala/org/antipathy/scoozie/action/ShellAction.scala

```diff
@@ -1,16 +1,16 @@
 package org.antipathy.scoozie.action
 
-import org.antipathy.scoozie.action.prepare.Prepare
-import org.antipathy.scoozie.configuration._
-import scala.xml.Elem
-import org.antipathy.scoozie.configuration.Credentials
-import scala.collection.immutable._
 import com.typesafe.config.Config
-import org.antipathy.scoozie.builder.{ConfigurationBuilder, PrepareBuilder}
-import scala.collection.JavaConverters._
-import com.typesafe.config.ConfigException
+import org.antipathy.scoozie.action.prepare.Prepare
+import org.antipathy.scoozie.builder.{ConfigurationBuilder, HoconConstants, PrepareBuilder}
+import org.antipathy.scoozie.configuration.{Credentials, _}
 import org.antipathy.scoozie.exception.ConfigurationMissingException
 
+import scala.collection.JavaConverters._
+import scala.collection.immutable._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)